pkg/importer/dataStream.go

```diff
@@ -112,6 +108,12 @@ type DataStreamOptions struct {
 	CertDir string
 	// InsecureTLS is it okay to skip TLS verification
 	InsecureTLS bool
+	// HasScratchAvailable indicates if we have an attached scratch space PVC to write to.
+	HasScratchAvailable bool
+	// ScratchDataDir is the path to the scratch space inside the container.
+	ScratchDataDir string
```

Comment:
How about getting rid of `HasScratchAvailable` and inferring with `ScratchDataDir != ""`?

Reply:
Done